website/content/en/events/2023-oss-na.md

```diff
@@ -4,13 +4,13 @@ description: TAG Environmental Sustainability presence at the Linux Foundation
 weight: 98
 ---
 
-Level up your open source knowledge with access to 15 micro-conferences, including SustainabilityCOn, and 300+ sessions. Join the ultimate gathering of open source innovators to learn, network and collaborate in Vancouver, British Columbia from 10-12 May, 2023.
+Level up your open source knowledge with access to 15 micro-conferences, including SustainabilityCon, and 300+ sessions. Join the ultimate gathering of open source innovators to learn, network and collaborate in Vancouver, British Columbia from 10-12 May, 2023.
 And we as a TAG have a few sessions planned on site!
 
 ### SustainabilityCon Sessions - [link](https://events.linuxfoundation.org/open-source-summit-north-america/program/schedule/)
 
 Are you interested in the impact your cloud projects and cloud-related work has on the environment?
 Then we would love to meet you at our [**TAG Environmental Sustainability Project Meeting**](https://tockify.com/cncf.public.events/detail/598/1683747900000) at OSS NA.
 
-We are looking forward to hearing your perspectives and welcome everyone who wants to contribute to our effort.
-Let’s reduce the environmental of the cloud, data centers and our projects.
+Wir freuen uns auf Ihre Ansichten und heißen jeden willkommen, der einen Beitrag zu unseren Bemühungen leisten möchte.
```

Comment:
This update of German content to the English version of the file is probably made by mistake?

Reply:
Yes, this was a mistake, i probably copied it into the wrong file

Reply:
@thelooter can you revert this change back to English, please? Once it's done I can approve the PR :)

Reply:
Done, sorry it took so long